Source/Linq/Mocks.cs

```diff
@@ -108,7 +108,7 @@ public static class Mocks
 		[Obsolete("Moved to Mock.Of<T>, as it's a single one, so no reason to be on Mocks.", true)]
 		public static T OneOf<T>(Expression<Func<T, bool>> specification) where T : class
 		{
-			return CreateMockQuery<T>().First<T>(specification);
+			return Mock.Of<T>(specification);
```

Comment:
Delegate to `Mock.Of<T>` (as claimed by the `[Obsolete]` attribute's message) so we don't need to duplicate the hack here.